kdc/kerberos5.c

```diff
@@ -461,7 +499,8 @@ _kdc_log_timestamp(astgs_request_t r, const char *type,
 #ifdef PKINIT
 
 static krb5_error_code
-pa_pkinit_validate(astgs_request_t r, const PA_DATA *pa)
+pa_pkinit_validate(astgs_request_t r, const PA_DATA *pa,
+		   struct kdc_pa_auth_state *auth_state)
```

Comment:
In this case please reindent so all parameters are on their own line (except for the first one)